src/ApparentHorizons/StrahlkorperInDifferentFrame.cpp

```diff
@@ -263,10 +293,61 @@ void strahlkorper_in_different_frame(
       center_dest);
 }
 
+template <typename SrcFrame, typename DestFrame>
+void strahlkorper_in_different_frame_aligned(
+    const gsl::not_null<Strahlkorper<DestFrame>*> dest_strahlkorper,
+    const Strahlkorper<SrcFrame>& src_strahlkorper, const Domain<3>& domain,
+    const std::unordered_map<
+        std::string, std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&
+        functions_of_time,
+    const double time) {
+  // Temporary storage; reduce the number of allocations.
+  Variables<
+      tmpl::list<::Tags::Tempi<0, 2, ::Frame::Spherical<SrcFrame>>,
+                 ::Tags::Tempi<1, 3, SrcFrame>, ::Tags::TempI<2, 3, SrcFrame>,
+                 ::Tags::TempI<3, 3, DestFrame>, ::Tags::TempScalar<4>>>
+      temp_buffer(src_strahlkorper.ylm_spherepack().physical_size());
+  auto& src_theta_phi =
+      get<::Tags::Tempi<0, 2, ::Frame::Spherical<SrcFrame>>>(temp_buffer);
+  auto& r_hat = get<::Tags::Tempi<1, 3, SrcFrame>>(temp_buffer);
+  auto& src_cartesian_coords = get<Tags::TempI<2, 3, SrcFrame>>(temp_buffer);
+  auto& dest_cartesian_coords = get<Tags::TempI<3, 3, DestFrame>>(temp_buffer);
+  auto& radius = get<::Tags::TempScalar<4>>(temp_buffer);
+
+  StrahlkorperTags::ThetaPhiCompute<SrcFrame>::function(
+      make_not_null(&src_theta_phi), src_strahlkorper);
```

Comment:
These tag functions are free functions (`StrahlkorperFunctions::theta_phi` here).  Call those instead of going through the tags.

Reply:
Done!